velox/vector/tests/VectorToStringTest.cpp

```diff
@@ -164,6 +164,12 @@ TEST_F(VectorToStringTest, constant) {
   ASSERT_EQ(
       nullConstant->toString(true), "[CONSTANT INTEGER: 100 elements, null]");
 
+  // Null constant of UNKNOWN type.
```

Comment:
Could you add to test toString() with index as well? Thanks!

Reply:
Good idea. Added.